freqtrade/pairlist/AgeFilter.py

```diff
@@ -23,6 +23,16 @@ def __init__(self, exchange, pairlistmanager,
         super().__init__(exchange, pairlistmanager, config, pairlistconfig, pairlist_pos)
 
         self._min_days_listed = pairlistconfig.get('min_days_listed', 10)
+
+        if self._min_days_listed < 1:
+            self.log_on_refresh(logger.info, "min_days_listed must be >= 1, "
```

Comment:
OperationalException should be raised here instead of logging (as for validation of parameters implemented in `__init__` of other Pairlist Handlers...

(same a few lines below)